### ISSUE TITLE
Transpiler tests fail on windows

### ISSUE THREAD

**Issue by rmtmckenzie** [cincheo/jsweet#215]
I'm trying to compile a project and the current release is missing some functionality that appears to be in the master branch of jsweet. 

While I realize it might not be 100% stable, I thought it was worth a shot trying out the most recent changes. But the transpiler is failing its tests when running on windows. I'm using oracle JDK 1.8.0_121 64 bit, have a recent npm and node versions installed (if that's relevant). It didn't fail on linux so I'm just going to use the build from that for now, but I thought it was worth mentioning as users of jsweet may run into some of the problems on windows only.

This is the output:
```
  OverloadTests.testWrongOverload:62->AbstractTest.eval:183->AbstractTest.eval:189->AbstractTest.eval:195->AbstractTest.eval:214->lambda$testWrongOverload$1:64 expected:<1,2,3,4,5,6,7> but was:<null>
  StructuralTests.testNoNameClashesWithFields:75->AbstractTest.eval:183->AbstractTest.eval:189->AbstractTest.eval:195->AbstractTest.eval:218 exception occured while running test StructuralTests.testNoNameClashesWithFields with module kind none
  SyntaxTests.testLiterals:189->AbstractTest.eval:183->AbstractTest.eval:189->AbstractTest.eval:195->AbstractTest.eval:218 exception occured while running test SyntaxTests.testLiterals with module kind none
```

And I've attached the failed tests:
[failed tests.zip](https://github.com/cincheo/jsweet/files/735096/failed.tests.zip)

I spent some time trying to figure out what was going on but not being familiar with the transpiler I didn't get very far.... At least some of the failed tests seem to be to do with unicode support.

**Comment by lgrignon** [comment]
Seems related to #157 and #103 
Could you please try to run those tests under Eclipse?

Thanks for reporting.

**Comment by lgrignon** [comment]
I think we finally succeeded to fix this annoying issue in version 2.0.0-SNAPSHOT (commit ca873ef1cfee2aef8cf09ad38985af4385128c19)

**Comment by lgrignon** [comment]
Fixed for 1.2.1 through df2286be5d7ca29dbb968b252ae06317137ea443 and deployed
